spec/project-spec.coffee

```diff
@@ -13,6 +13,39 @@ describe "Project", ->
   beforeEach ->
     atom.project.setPaths([atom.project.getDirectories()[0]?.resolve('dir')])
 
+  describe "constructor", ->
+    it "tries to update repositories when a new RepositoryProvider is registered", ->
+      atom.project.setPaths(["/tmp"])
```

Comment:
This shouldn't use a hard-coded platform-specific path, it should use something like `temp.mkdirSync('atom-project')` instead.